src/components-examples/material/autocomplete/autocomplete-overview/autocomplete-overview-example.html

```diff
@@ -7,7 +7,7 @@
            [formControl]="stateCtrl">
     <mat-autocomplete #auto="matAutocomplete">
       <mat-option *ngFor="let state of filteredStates | async" [value]="state.name">
-        <img class="example-option-img" aria-hidden [src]="state.flag" height="25">
+        <img alt="" class="example-option-img" [src]="state.flag" height="25">
```

Comment:
An empty `alt` means the image is decorative and ignored by non-visuals browsers: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt

Based on my understanding (and conversation with @twerske), this should be the more correct way of doing it since it maintains `role="img"` whereas `aria-hidden` does not.

Reply:
Ah okay, TIL.